common/app/html/HtmlPage.scala

```diff
@@ -42,8 +41,8 @@ object HtmlPageHelpers {
     )
   }
 
-  def FaciaCSSFile: String = if (PillarCards.isSwitchedOn) "facia.pc" else "facia"
-  def ContentCSSFile: String = if (PillarCards.isSwitchedOn) "content.pc" else "content"
-  def RichLinksCSSFile: String = if (PillarCards.isSwitchedOn) "rich-links.pc" else "rich-links"
+  def FaciaCSSFile(implicit request: RequestHeader): String = if (mvt.Garnet.isParticipating) "facia.pc" else "facia"
```

Comment:
i can\t work out how to get the request header here, do i have to add it to everything in the call chain above it?

Reply:
Basically yes. Let me know if you need help to do so